### DISCUSSION TITLE
Interesting feedback on the mailing list

### DISCUSSION THREAD

**Discussion by tobbi** [SuperTux/supertux#2583] [Ideas]
deathaster95 wrote some interesting feedback including things to improve SuperTux on the mailing list, here:
http://lists.lethargik.org/pipermail/supertux-devel-lethargik.org/2014-April/003761.html

We should filter it and see which of this is suitable for post-0.4.0

**Comment by Karkus476** [comment]
Let's email him back, explaining that we are reviewing his ideas, and invite him to join us here (on GitHub). Anyone interested in designing levels would be great!

**Comment by Karkus476** [comment]
I like the belly glide idea, this could work very well with slopes

**Comment by Mathnerd314** [comment]
It's been an idea for a while, old wiki page: http://supertux.lethargik.org/wiki/Slider
Even made it into Grumbel's page: http://supertux.lethargik.org/wiki/Milestone_2_Design_Document/Actions#Sliding_down_Slopes

**Comment by maxteufel** [comment]
Copy-paste in case lethargik.org goes down:

> (Is this how I am supposed to feedback? I am not quite sure, your devel page confused me a bit)
> I will just say that I've known Supertux for like 5 years know, and it's been part of my childhood, really. I am 18 years now, by the way.
> 
> I think it's a great game, with a lot of potential (and great artwork, really. I've been drawing Tux and stuff a lot when I was younger), and I have a bit of knowledge of game programming, level design etc. I love everything about Supertux, and I think every single one of you did an awesome job on the game! _Applause_
> 
> Well, now for my feedback/ideas, to make the game even MORE awesome:
> 
> 1)Bring the forest level back (because it looked so awesome! When I was younger and I finished Supertux, I searched around the Internet a bit and I found a picture of Supertux, in a forest level! It got me so excited, because it seemed so cool! Later I learned that you ACTUALLY put that in the game, and when I played the forest levels, I fell in love with them.).
> If you plan to bring the forest back soon, you could make it an exact opposite of the ice levels, which have a lot of straight-forward, horizontal levels (like Super Mario Bros.) : You could make Tux jump around trees a lot more, and make vertical-levels (like Ice Climber/Mario vs. Donkey Kong)! It'd create a nice contrast, fit the mood of a forest (you know, with huge trees and so on), and you could even include mountains or something like that!
> 2)The backflip is neat, but I never got to use it really, except when I needed to jump on an itembox. It's a good, new idea, and you could combine it with something else (Ninja-kick when you press CTRL while doing the backflip? Sonic-like roll? Anything!), because then it could be a new attack, and these attacks could be used against certain enemies (Ninja-Snowball maybe?)
> 3) The Butt-stomp is a good idea too, but it only works sometimes, and it is no difference from the normal jump...You could change it so it does twice the damage (for example, killing these tree-enemies in 1 stomp instead of 2), and if you do that, you should make it harder to perform:
> Maybe you can only make a butt-stomp if you do a double-jump? (See point 5 for that) Maybe you can only do it if you jump, then press "Down" 2 times? Something like that! And maybe there is an enemy that can only be defeated by a butt-stomp, just like an enemy that can only be defeated by a backflip!
> 4) You NEED to make Tux glide on his belly, like a penguin would...It would be so much fun, to race down hills on his belly, smashing through enemies, just like in Tux Racer! You could make an entire level based on that mechanic! Like that you have to escape from a giant snowball by surfing down an icy mountain! Also, rememberthe penguins of Madagascar? They made penguins cool! But you existed BEFORE them, so you have the right to take something from them! They can do karate and belly-glides? WELL SO CAN TUX NOW. Thank you very much dreamworks, but now it's OUR turn for making penguins awesome again.
> 
> 5) More items, for more actions! Like an item that you can do a double jump! Or maybe even fly, like the Tanooki-Suit of SMB 3! You could have certain enemies that can only be defeated by a certain item! That nail-enemy? Well he can only die if you throw a hammer at him! And maybe if you nail him down in a certain spot in the level, you can reach a secret area! 
> 
> Also: I think that the fire-flower should have more actions: It's fire! It should melt ice blocks. If you put ice blocks in the first world, you can have so many possibilities for level design! Maybe there is an frozen pipe that hides a secret area! Or the key to the level exit is frozen in ice blocks somewhere! 
> I also think that the first world, the ice-world, should have more ice-flowers in it, because well, it's an ice world! And that the forest-world has more fire-flowers in it. While I am at it, the ice flower could have more actions too: For example, it could make freeze flying enemies, so you can build yourself an icy bridge to an unreachable area! Or it could create ice blocks on water, so you can walk to certain places you couldn't normally reach!
> 
> 6) Less checkpoints, because the game is still incredibly easy in the later levels. You should just balance the difficulty a little bit, so that it goes from easy to hard, because it is not always the case. But maybe I am just used to Super Mario games, and they're easyto me in general.
> 
> 7) More stuff on the world map! Right now, it is mostly just walking from level A to level B, and then to level C. You could make it like in Super Mario Bros 3, that you have to break stones to get a different path, or include pipes to warp around the map a bit! You could make an entire world that is just a giant maze! Maybe the final world, Nolok's castle? 
> 8) Some enemies don't make a lot of sense...the sleeping spike-snowball is sleeping, but why? He doesn't do much and still dies in one hit, unlike the hedgehog, who is just an enemy that is hard to kill (turns when you throw a fireball at him, so rude!). Also, it's been long since I saw the fish enemies! They could swim around in the water (so you can see them), and when you're near them, they JUMP AT YOU. Do you know the game "Ecco the dolphin"? They could behave like the dolphin in the game, that they jump out of the water in an arch.
> Also, the poison plants in the forest level are confusing, because at first I thought they were just part of the scenery, until they started hurting me. You could animate them, that they wiggle around a little, so people at least see them!
> 
> 9)As much as you love the Homer Simpson yeti, I like the other version of him more (shown in the concept art), because it made him look much more realistic, which fit the artstyle of the game, and because of that both beautiful and disturbing, dangerous.
> 
> The rest is excellent though, and these are just suggestions, but I would love to see them in the game, because they fit the mood, and make the game so much more fun :) Thanks for reading (if you did), I would love to play the next version of SuperTux soon.
> 
> P.S.: The idea of the lamp-item, which makes invisible blocks appear, and that you have to combine red and blue lamps to make purple blocks appear is a genius idea. I've never seen anything like that in a game before
